grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -177,11 +177,12 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
                     <AttachedIncidentsList id={incident.pk} getUnattachClickHandler={this.getUnattachClickHandler} />
                   </div>
                   <div className={cx('column')}>
-                    <VerticalGroup>
-                      <PagedUsers
-                        pagedUsers={incident.paged_users}
-                        onRemove={this.handlePagedUserRemove}
-                        disabled={incident.is_restricted}
+                    <VerticalGroup style={{ display: 'block' }}>
```

Comment:
@grafana/grafana-oncall-frontend how can I get these two vertically aligned elements to have the same width (the borders don't line up on the right-hand side):

![Screenshot 2023-10-20 at 14 56 52](https://github.com/grafana/oncall/assets/9406895/e1fe5a63-1147-439b-902b-a9c017586e25)

Reply:
Can hop on a quick call if you need help on this one. Maybe there's an additional padding/margin come from an inside element

Reply:
ahh this comment is outdated, I figured this out 😄